Update flush policies to include overriding standard policies (#6473)
segmentio/segment-docs: src/connections/sources/catalog/libraries/mobile/react-native/index.md

- `CountFlushPolicy` triggers when you reach a certain number of events
- `TimerFlushPolicy` triggers on an interval of milliseconds
- `StartupFlushPolicy` triggers on client startup only

> > info ""
> > Implementing custom flush policies will replace the default Count and Timer policies provided by Segment. To incorporate custom policies, ensure to explicitly add the Timer and Count policies to the client's Flush Policies configuration if needed.

Reviewer: ```suggestion

> info ""
> If you implement custom flush policies, they replace Segment's default Count and Timer policies. To incorporate custom policies, add your custom Timer and Count policies to the client's Flush Policies configuration.

```
